### PR TITLE
Update README, add laravel-broadcaster note

### LINKED ISSUE #30: Add a note pointing to new laravel-broadcaster

### PR DESCRIPTION
**Pull request by sacOO7**
- Fixes #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Ably PHP Laravel Wrapper
+# Simple Ably-PHP Laravel Wrapper
```
Comment (owenpearson) [inline README.md:1]: I'm not sure if I like this, I think the title should stay the same and instead we should use markdown to add a visible warning just below the h1, like:

> **Warning**
> If you're using Laravel and want to support **realtime broadcasting and events**, it is recommended to use [ably/laravel-broadcaster](https://packagist.org/packages/ably/laravel-broadcaster/) instead.

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sacOO7) [reply README.md]: Updated!

File: README.md [hunk 2]
```diff
@@ -19,9 +22,15 @@ We regression-test the library against a selection of PHP versions (which will c
...
+## Note
+
+If you're using Laravel and want to support **realtime broadcasting and events**, you may want to check out [laravel-broadcaster](https://packagist.org/packages/ably/laravel-broadcaster/).
+
 ## Known Limitations
 
-Currently, this SDK only supports [Ably REST](https://www.ably.com/docs/rest). However, if you want to subscribe to events in PHP, you can use the [MQTT adapter](https://www.ably.com/docs/mqtt) to implement [Ably's Realtime](https://www.ably.com/docs/realtime) features in PHP. 
+- Currently, this SDK only supports [Ably REST](https://www.ably.com/docs/rest). However, if you want to subscribe to events in PHP, you can use the [MQTT adapter](https://www.ably.com/docs/mqtt) to implement [Ably's Realtime](https://www.ably.com/docs/realtime) features in PHP. 
+
+- This wrapper has limited usecases and [laravel-brodcaster](https://github.com/ably/laravel-broadcaster) is recommended for most cases.
```
Comment (qsdstefan) [inline README.md:33]: brodcaster -> broadcaster

**Review by qsdstefan — CHANGES_REQUESTED** [review]
Please fix the typo and I'll happily approve

File: README.md (hunk 2 above)
Comment (sacOO7) [reply README.md]: updated!

**Review by owenpearson — APPROVED** [review]
LGTM

**Pull request merged**